### PR TITLE
Correcting the Gaussian Formula

### PR DESCRIPTION
**Pull request by aryan26roy**
I have added the parenthesis around the 2*sigma^2 term.

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
None of the doctests needed to change?!?  That implies that the results of the calculation did not change.  This needs some doctests which fail under the old formulation and pass under the new formulation.

**Comment by aryan26roy** [comment]
I tried the doctests provided in the code and the new formulation provides exactly the same results.

**Comment by cclauss** [comment]
So this PR _corrects_ an algorithm that was already correct?

**Comment by aryan26roy** [comment]
It turns out when you vary the value of mu, the differences emerge. I have added a doctest which the old formulation would fail. Should i add more?

**Comment by spamegg1** [comment]
@aryan26roy Also try varying the sigma #2212

**Comment by aryan26roy** [comment]
Hey, i have varied the sigma as well. The old formulation will fail the two new doctests. However, it is encountering a build error.

**Comment by cclauss** [comment]
Whitespace errors that should be easy to fix.
https://travis-ci.com/github/TheAlgorithms/Python/builds/177610419#L252

**Comment by aryan26roy** [comment]
Hey, i have removed the error.

File: maths/gaussian.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def gaussian(x, mu: float = 0.0, sigma: float = 1.0) -> int:
     
     >>> gaussian(1,4,3)
     0.0806569081730478
-
+
```
Comment (cclauss) [inline maths/gaussian.py]: whitespace error.

Comment (aryan26roy) [reply maths/gaussian.py]: That is an older version. I have updated and removed all the white spaces.

File: maths/gaussian.py [hunk 2]
```diff
@@ -15,13 +15,13 @@ def gaussian(x, mu: float = 0.0, sigma: float = 1.0) -> int:
...
-    
+
     >>> gaussian(1,4,2)
     0.06475879783294587
-    
+
     >>> gaussian(1,5,3)
     0.05467002489199788
-    
+
```
Comment (aryan26roy) [inline maths/gaussian.py:24]: See?

**Pull request merged**